core/state.go

```diff
@@ -13,78 +13,78 @@ import (
 
 // Operators
 
-// OperatorSocket is formatted as "host:dispersalPort;retrievalPort;v2DispersalPort"
-type OperatorSocket string
+// OperatorSocket is formatted as "host:dispersalPort;retrievalPort;v2DispersalPort;v2RetrievalPort"
+type OperatorSocket struct {
+	host            string
+	v1DispersalPort string
+	v1RetrievalPort string
+	v2DispersalPort string
+	v2RetrievalPort string
+}
 
 func (s OperatorSocket) String() string {
```

Comment:
Marshal/Unmarshal or Serialize/Deserialize? String() feels like producing something for logging, debugging or display.

Reply:
Ack. Good call.

Reply:
Thinking more about this, a Marshal/Serialize  and Unmarshal/Deserialize might not be accurate because what we want is the socket string representation. Plus we need only the stringified output, without converting this to serializables, i.e there is no need for implementing deserialize unmarshals.

Was originally overriding the String() method. To avoid confusion for logging, debug use case. I will call this `Normalize` instead, as we are using custom logic to represent socket string in different runtime modes.